app/controllers/media_objects_controller.rb

```diff
@@ -70,8 +70,9 @@ def destroy
 
   # POST /media_object/saveMedia
   def saveMedia
-    extension = params[:upload].original_filename.split('.')[1]
-    unless FileUploader.upload_whitelist.include? extension
+    extensions = params[:upload].original_filename.split('.')
```

Comment:
``extension = params[:upload].original_filename.split('.')[-1]``
Is what you're looking for